### ISSUE TITLE
ndb doesn't start (ECONNRESET)

### ISSUE THREAD

**Issue by romgrk** [GoogleChromeLabs/ndb#240]
### Steps to reproduce

**Tell us about your environment:**

* ndb version: 1.0.27 and up don't work. 1.0.26 works
* Platform / OS version: Ubuntu 18.10
* Node.js version: 11.10.0

**What steps will reproduce the problem?**

1. `npm i -g ndb@1.0.27`
2. `ndb .`

**What is the expected result?**
ndb starts.

**What happens instead?**
```
events.js:193
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at _errnoException (util.js:1031:13)
    at Pipe.onread (net.js:619:25)
```

**Comment by felixdrp** [comment]
Have you try to close all your Chrome browser processes and start ndb?
It let me start ndb when Chrome is closed.Issue #239

**Comment by ak239** [comment]
One possible root of the issue is resolved in `1.0.47`. Fix for another is coming soon (https://github.com/GoogleChromeLabs/ndb/pull/260). I will close this one for now as duplicate of https://github.com/GoogleChromeLabs/ndb/issues/248 .
